scripts/build-renode.sh

```diff
@@ -75,12 +75,25 @@ RENODE_RESC="$RENODE_SCRIPTS_DIR/litex_buildenv.resc"
 RENODE_REPL="$RENODE_SCRIPTS_DIR/litex_buildenv.repl"
 
 mkdir -p $RENODE_SCRIPTS_DIR
-python $TOP_DIR/third_party/litex-renode/generate-renode-scripts.py $LITEX_CONFIG_FILE \
-	--repl "$RENODE_REPL" \
-	--resc "$RENODE_RESC" \
-	--bios-binary "$TARGET_BUILD_DIR/software/bios/bios.bin" \
-	--firmware-binary "$TARGET_BUILD_DIR/software/$FIRMWARE/firmware.bin" \
-	--configure-network ${TAP_INTERFACE:-""}
+
+if [ "$FIRMWARE" == "linux" ]; then
+	python $TOP_DIR/third_party/litex-renode/generate-renode-scripts.py $LITEX_CONFIG_FILE \
+		--repl "$RENODE_REPL" \
+		--resc "$RENODE_RESC" \
+		--bios-binary "$TARGET_BUILD_DIR/software/bios/bios.bin" \
```

Comment:
I'm pretty sure the values below shouldn't be hard coded?

Reply:
Those offsets are currently hardcoded in the LiteX BIOS - https://github.com/enjoy-digital/litex/blob/master/litex/soc/software/bios/boot.c#L411-L414. They are relative to the `FLASH_BOOT_ADDRESS` and are independent of the platform.

We would have to rework LiteX to export those offsets in the configuration file (`csr.csv`) if we want to avoid hardcoding it in this script.

Reply:
Can we extract them out of the `boot.c` file?

Reply:
I proposed changes to LiteX that will allow to put those offsets into the configuration file and be dynamically parsable: https://github.com/enjoy-digital/litex/pull/389

Reply:
Moved to: https://github.com/timvideos/litex-buildenv/pull/340